src/prefect/infrastructure/docker.py

```diff
@@ -226,6 +231,22 @@ class DockerContainer(Infrastructure):
         default=True,
         description="If set, the output will be streamed from the container to local standard output.",
     )
+    memswap_limit: Union[int, str] = Field(
+        default=None,
+        description="Total memory (memory + swap), -1 to disable swap.",
```

Comment:
What's the default Docker behavior for this one?

Reply:
Docker does not set a `mem_limit` by default, as explained here: https://codefresh.io/blog/docker-memory-usage/#:~:text=Docker%20does%20not%20apply%20memory,consume%20the%20entire%20host's%20memory.

`memswap_limit` only has meaning if `mem_limit` is set, see: https://docs.docker.com/config/containers/resource_constraints/#--memory-swap-details
If `mem_limit` is set, `memswap_limit` defaults to allowing the container to use as much swap as memory.

I've improved the docs of `DockerContainer` to explain this behavior.